geopy/distance.py

```diff
@@ -15,13 +15,29 @@
 :const:`distance.EARTH_RADIUS`, so it can be customized (it should
 always be in kilometers, however).
 
-Vincenty distance (:class:`.vincenty`) uses a more accurate ellipsoidal model
```

Comment:
But Vincenty is still there, right? And it's the default `distance` when geodesic is not available (a pretty common use-case, I believe). So I suggest to return back this part of doc.